lib/reporters/mini.js

```diff
@@ -322,13 +323,24 @@ class MiniReporter {
 
 		if (evt.err.stack) {
 			const {stack} = evt.err;
-			if (stack.includes(os.EOL)) {
+			if (stack.includes('\n')) {
 				this.lineWriter.writeLine();
-				this.lineWriter.writeLine(colors.errorStack(stack));
+				this.lineWriter.writeLine(this.formatErrorStack(stack));
 			}
 		}
 	}
 
+	formatErrorStack(stack) {
+		return stack.split('\n')
+			.map(line => {
+				if (StackUtils.nodeInternals().some(internal => internal.test(line))) {
```

Comment:
`nodeInternals()` allocates a new array every time it's called. Let's assign it to a variable instead.